src/window.rs

```diff
@@ -703,7 +703,7 @@ impl Window {
     ///
     /// **iOS:** Can only be called on the main thread.
     #[inline]
-    pub fn available_monitors(&self) -> AvailableMonitorsIter {
+    pub fn available_monitors(&self) -> impl Iterator<Item = MonitorHandle> {
         let data = self.window.available_monitors();
         AvailableMonitorsIter {
```

Comment:
We should remove the `AvailableMonitorsIter` type since it is no longer required. Then this function can simply be `self.window.available_monitors().into_iter().map(|inner| MonitorHandle { inner })`.

Reply:
Good idea :)